internal/eventingester/store/insertion.go

```diff
@@ -18,20 +18,20 @@ import (
 // InsertEvents takes a channel of armada events and inserts them into the event db
 // the events are republished to an output channel for further processing (e.g. Ackking)
 func InsertEvents(ctx context.Context, db EventStore, msgs chan *model.BatchUpdate, bufferSize int,
```

Comment:
This piece of code should probably refactored to have a struct storing some common stuff (e.g. list of fatal errors).  I didn't do it as part of this change, just because I wanted to get this in relatively quickly.